dvc/remote/base.py

```diff
@@ -173,7 +173,21 @@ def changed(self, path_info, checksum_info):
         return False
 
     def save(self, path_info, checksum_info):
-        raise RemoteActionNotImplemented("save", self.scheme)
+        if path_info["scheme"] != self.scheme:
```

Comment:
Are we currently testing for this case?

Reply:
@mroutis Unfortunately we don't. These are mostly placeholders just in case something goes wrong. Will cover it will a unit test while working on improving our coverage. Thank you! :slightly_smiling_face: